gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go

```diff
@@ -361,7 +361,7 @@ func (p *glusterBlockProvisioner) createVolume(volSizeInt int, blockVol string,
 		// Execute gluster-block command.
 		cmd := exec.Command(
 			config.opMode, "create", config.blockModeArgs["glustervol"]+"/"+blockVol,
-			"ha", haCountStr, config.blockModeArgs["hosts"], sizeStr, "--json")
+			"ha", haCountStr, config.blockModeArgs["hosts"], sizeStr+"GB", "--json")
```

Comment:
do you need to check if sizeStr already has `G` or `GB` in it?

Reply:
Actually its not required as the sizestr is derived by `roundupsize` which use the value of `resource request` in bytes. Thanks